public/locales/sw/menu-drawer.json

```diff
@@ -4,7 +4,7 @@
     "iframeTitle": "Sanidi arifa zako za EarthRanger"
   },
   "betaToggles": {
-    "patrolUI": "UI Mpya wa Fomu ya Doria",
+    "playSoundForAlarmRadioState": "Cheza sauti na onyesha arifa wakati hali ya redio inabadilika kuwa nyekundu (kengele)",
```

Comment:
Just out of curiosity, Where did you get all those new translations? using just a regular online translator?

I'm asking just to be aware of the process We are doing in this scenario

Reply:
For english, spanish and french I have confidence. For the others, I used chatGPT, since I think it understands more the context of the text than a simple translator